### DISCUSSION TITLE
Link to Russian music channel

### DISCUSSION THREAD

**Discussion by johnearl3** [streamlink/streamlink#4296] [Q&A]
I'm looking for a channel called ritsatv ru.
The link is here to watch - https://www.twitch.tv/ritsatv - 
I would like to watch the channel on VLC Media Player or Duplex Play TV App
Cant find a stable link for this channel - the only link that is stable is https://www.twitch.tv/ritsatv - is there anyway I can get the link from twitch tv to put in vlc or duplex play tv app.

**Comment by bastimeyer** [comment]
> Cant find a stable link for this channel
> twitch.tv/ritsatv

What do you mean? This is the canonical (and thus "stable") link of the channel.
If you're talking about HLS stream URLs on Twitch, then these are dynamic, because they depend on a mandatory access token that needs to be requested from their API.

> is there anyway I can get the link from twitch tv to put in vlc or duplex play tv app.

Streamlink lets you output the resolved HLS stream URL via `--stream-url`:
https://streamlink.github.io/cli.html#cmdoption-stream-url

```sh
streamlink --stream-url 'twitch.tv/ritsatv' best
```

This shouldn't be necessary though if you only want to watch the stream in your local video player.

Your question makes it sound like you stumbled upon this repository by accident without knowing what Streamlink actually is.
- https://streamlink.github.io/index.html
- https://streamlink.github.io/cli.html
